services/shop/src/index.js

```diff
@@ -16,7 +16,15 @@
 import express from 'express';
 import session from 'express-session';
 import MemoryStoreFactory from 'memorystore';
-import {DSP_HOST, EXTERNAL_PORT, PORT, SHOP_DETAIL, SHOP_HOST} from './env.js';
+import {
+  DSP_HOST,
```

Comment:
I don't have a better idea to solve this....but somehow it is confusing to talk about DSP_HOST, followed by DSP_A_HOST and DSP_B_HOST. 

Maybe DSP_COMPONENT_A_HOST or anything like that?

Reply:
Hmm. Severin and I have been talking about separating the top-level seller from the ad server and implementing a separate top-level seller. I don't want to just change these constants because that convention of "DSP-A" and "SSP-B" is used through the codebase and they should all be updated. 

Let me look into improving the service names when we tackle the top-level seller task.